scripts/manage/restore_backup_to_cloud_shell.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+bold() {
+  echo ". $(tput bold)" "$*" "$(tput sgr0)";
+}
+
+~/spinnaker-for-gcp/scripts/manage/check_git_config.sh || exit 1
+
+while getopts ":p:r:h:" options; do
+  case $options in
+    p ) PROJECT_ID=$OPTARG ;;
+    r ) CONFIG_CSR_REPO=$OPTARG ;;
+    h ) GIT_HASH=$OPTARG ;;
+    \? ) bold "invalid option supplied: -$OPTARG"
+  esac
+done
+
+EXAMPLE_COMMAND="'restore_backup_to_cloud_shell.sh -p PROJECT -r REPOSITORY_NAME -h GIT_HASH'"
```

Comment:
If I just copy/paste and run this command, there is no indication that `GIT_HASH` wasn't recognized as an actual commit hash. Same if I enter any other random string in place of a legitimate hash. We should confirm that the hash refers to an actual commit and print an error message if it does not.